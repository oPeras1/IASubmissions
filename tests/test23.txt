1	2	2	2	2	3	3	4	4	4	5	5	6	6	7	8	9	9	9	9
1	1	2	2	10	3	3	11	4	5	5	6	6	7	7	8	12	12	13	9
1	14	14	15	10	10	3	11	11	11	5	6	7	7	16	8	12	12	13	13
1	14	15	15	10	10	3	3	17	11	5	6	5	5	16	8	8	12	13	13
14	14	15	15	10	10	10	17	17	17	5	6	5	16	16	18	18	12	13	19
14	15	15	20	10	17	17	17	17	17	5	5	5	16	21	21	18	12	12	19
14	20	20	20	22	22	22	22	22	17	17	21	21	16	21	18	18	21	19	19
20	20	23	20	24	24	22	22	25	17	21	21	21	21	21	18	21	21	26	26
23	23	23	24	24	25	25	25	25	17	21	27	27	27	21	21	21	21	21	26
23	28	25	25	25	25	17	17	17	17	17	27	29	21	21	30	26	26	26	26
28	28	28	28	28	28	31	31	31	32	32	27	29	29	29	30	26	33	33	34
28	35	35	36	36	31	31	31	37	37	32	32	29	38	39	30	30	33	33	34
28	35	36	36	36	40	41	37	37	32	32	38	38	38	39	30	30	33	34	34
42	35	36	35	36	40	41	41	41	41	41	38	39	39	39	30	39	39	34	34
42	35	35	35	43	40	40	41	41	41	38	38	39	44	39	39	39	39	34	45
42	43	43	43	43	40	46	46	47	38	38	38	44	44	39	48	48	48	45	45
42	42	49	49	49	40	46	47	47	50	38	51	51	44	39	48	52	52	52	45
42	42	49	46	46	46	46	47	47	50	38	51	51	51	39	48	52	52	52	52
42	49	49	49	46	53	53	53	47	50	38	51	54	51	55	55	55	52	52	56
42	46	46	46	46	53	53	47	47	50	38	54	54	54	54	55	56	56	56	56