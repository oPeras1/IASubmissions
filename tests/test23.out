T	2	L	L	L	S	3	T	T	T	5	5	6	6	S	8	I	I	I	I
T	T	L	2	10	S	S	11	T	5	5	L	L	S	S	L	12	12	S	9
T	14	14	T	10	10	S	L	L	L	5	L	7	S	16	L	12	I	S	S
1	S	T	T	S	S	3	3	17	L	5	L	5	5	S	L	L	I	13	S
S	S	15	T	10	S	S	T	T	T	L	6	5	S	S	18	18	I	13	L
S	15	15	L	10	17	17	17	T	17	L	L	L	S	21	21	S	I	12	L
14	L	L	L	22	I	I	I	I	17	17	21	21	16	21	S	S	21	L	L
20	20	23	20	S	S	22	22	L	17	21	21	21	21	21	S	21	21	26	26
L	L	L	S	S	25	L	L	L	17	21	L	L	L	I	I	I	I	21	L
L	28	25	25	25	25	17	17	17	17	17	L	29	21	21	30	26	L	L	L
28	28	I	I	I	I	L	L	L	32	T	27	L	L	L	T	26	33	S	34
28	L	L	36	T	31	L	31	S	S	T	T	L	38	39	T	T	S	S	L
28	L	36	T	T	40	41	S	S	32	T	38	38	38	L	T	30	S	34	L
42	L	36	35	T	40	41	41	T	41	41	S	L	L	L	30	39	39	L	L
42	35	35	35	43	L	L	T	T	T	S	S	39	T	39	39	39	39	34	T
42	I	I	I	I	L	46	46	S	38	S	38	T	T	39	L	L	L	T	T
I	42	L	49	49	L	46	S	S	I	38	L	51	T	39	L	52	52	52	T
I	42	L	46	L	46	46	S	47	I	38	L	L	L	39	48	I	I	I	I
I	L	L	49	L	T	T	T	47	I	38	51	T	51	T	T	T	52	52	L
I	46	46	L	L	53	T	47	47	I	38	T	T	T	54	T	56	L	L	L